manifests/4.2/sriov-network-operator-sriovnetwork.crd.yaml

```diff
@@ -44,6 +40,8 @@ spec:
               type: boolean
             trust:
               type: boolean
+              minimum: 0
```

Comment:
why adding min/max for boolean type value?

Reply:
It is a error caused by rebase, I'll fix it